Asterix/loop/correction_loop.py

```diff
@@ -155,7 +146,7 @@ def correction_loop(testbed: OptSy.Testbed,
     return Resultats_correction_loop
 
 
-def correction_loop_1matrix(testbed: OptSy.Testbed,
+def correction_loop_1matrix(testbed: Testbed,
```

Comment:
I personally don't think we need type hints at all in Asterix (specifying the type of a function variable in the function signature). It's hard enough to enforce proper docstrings, and for all goals and purposes of Asterix, type hints add zero benefit, so I would just start dropping them altogether.

Reply:
I disagree, I started using that because if you indicate those, vscode allow me to use autocomplete with all the testbed methods, it's super useful :-) 
For int, or floor I agree it's useless, and I don't indicate it, but for complex classes, its awesome !

Reply:
I see. Learned something new!